package-lock.json

```diff
@@ -5,6 +5,7 @@
   "requires": true,
   "packages": {
     "": {
+      "name": "mattermost-webapp",
```

Comment:
Is this change intended? Seems like the kind of change that different versions of npm would add/remove between PRs.

Reply:
Not really for this, but I didn't want to bother submitting a separate PR for that.

It keeps getting added and removed periodically. I think it's either caused by someone's NPM  being the wrong version or a difference between `npm install` and `npm add` again.